yurt/yurt_core/setup.py

```diff
@@ -52,19 +52,19 @@ def enable_git_repo(config_settings, project_name):
     os.chdir(current_path)
 
 
-def add_all_files_to_git_repo(_, project_name):
+def add_all_files_to_git_repo(*args):
```

Comment:
if we're going to go the `*args` route instead of a class based version, I think you need to set the values to variables at the top of the function to make this more readable - right now down below it's hard to know what `args[1]` is in the context of the code. If the first line of this function set `project_name = args[1]` then down below you used `project_name` that'd be great. This goes for all of these methods that are using `*args` now